code/modules/mob/mob_helpers.dm

```diff
@@ -2,7 +2,7 @@
 /proc/isxenomorph(A)
 	if(istype(A, /mob/living/carbon/human))
 		var/mob/living/carbon/human/H = A
-		return istype(H.species, /datum/species/xenos)
+		return istype(H.species, /singleton/species/xeno)
```

Comment:
you've changed the path here, that will cause issues. It should be /singleton/species/xenos